tools/build_script_generator/cmake/resources/ModmConfiguration.cmake.in

```diff
@@ -164,6 +162,8 @@ set({{ name | upper }}{{ "_" ~ (profile | upper) if profile | length else "" }}
 endfunction()
 
 function(modm_targets_create project_name)
+  find_package (Python3 COMPONENTS Interpreter REQUIRED)
```

Comment:
What's the difference? Does cmake has it's own package manager???

Reply:
Not quite a package manager, just a set of modules to find dependecies.
`find_program` fails to find a valid Python executable on my Windows machine and reading the docs it seems to only search a predefined set of paths per platform for a name match.
`find_package` in this case will use the FindPython3 module shipped with CMake to find python so it should be more flexible and does find my Python install.